book/modules/appendix1.tex

```diff
@@ -71,15 +71,29 @@
 		and the system of linear equations corresponding to the vector equation $t\vec w+\vec u=r\vec v$ where $\vec w=\mat{1\\-1}$, $\vec u=\mat{2\\3}$,
 		and $\vec v=\mat{4\\4}$.
 
-		XXX Finish
+		Paying attention to the coordinates and treating $x$, $y$, and $z$ as coefficients\index{Linear combination!coefficient of}, we can see that $\systeme{2x+3y+z=2,0x+y-z=-1}$ corresponds to the vector equation
+		\[
+			x\mat{2\\0} + y\mat{3\\1} + z\mat{1\\-1} = \mat{2\\-1}.
+		\]
+		
+		As for the vector equation $t\vec w+\vec u=r\vec v$, rewriting each vector in coordinates gives us a corresponding system of linear equations:
+		\begin{align*}
+			t\vec w + \vec u = r\vec v
+			\rightarrow
+			t\mat{1\\-1} + \mat{2\\3} &= r\mat{4\\4}\\
+			\mat{t+2\\-t+3} &= \mat{4r\\4r}
+			\rightarrow
+			\systeme{t+2=4r, -t+3=4r}.
+		\end{align*}
```

Comment:
We've defined SLEs to have constants on the right, so these equations need to be rearranged.